nifi-nar-bundles/nifi-asn1-bundle/nifi-asn1-services/src/test/resources/ASN1.xml

```diff
@@ -0,0 +1,1360 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
This needs an Apache header or to be added to the Rat exclusion list (I prefer the former)